docs/LOG_WRITER.md

```diff
@@ -1,54 +1,32 @@
+# LogWriter
 
+LogWriter takes care of deciding where to log the messages.
 
+### Prebuilt Loggers
 
+By default `Kermit` provides default `LogWriter` for each platform
 
+- `LogcatWriter` - Uses LogCat to send logs in Android
+- `CommonWriter` - Uses println to send logs in Kotlin
```

Comment:
nitpick, i'd keep the order that was there originally i.e. put `CommonWriter` first. Keeps a general->specific semantic and lines up with common being the default

Reply:
Done